ui/appui-react/src/appui-react.ts

```diff
@@ -86,6 +86,7 @@ export * from "./appui-react/hooks/useSolarDataProvider";
 
 export * from "./appui-react/imodel-components/spatial-tree/SpatialContainmentTree";
 export * from "./appui-react/imodel-components/category-tree/CategoriesTree";
+export * from "./appui-react/imodel-components/category-tree/CategoryVisibilityHandler";
```

Comment:
@bsteinbk this is marked as internal, what should i update this to be? The ModelsVisibilityHandler is marked as alpha.

Also, are you aware of any reason why Dan left these components in ui-framework/imodel-component here and not move them to imodel-components-react?

Reply:
@aruniverse I am not sure if Dan was behind these components, I think Grigas's team did the work for the Visibility trees. 
 @saskliutas may know more about this.  It would be safest to use internal unless there is a reason we want this exposed outside of Bentley.